src/components/input/__tests__/input-test.js

```diff
@@ -0,0 +1,184 @@
+/** @jsx React.DOM */
+var React = require('react');
+var InputView = require('../code/views/inputView.jsx');
+
+describe('InputComponent', function() {
+
+  it('should render text input', function() {
+    var input = TestUtils.renderIntoDocument(
+      <InputView type="text" />
+    );
+
+    var renderedInput = TestUtils.findRenderedDOMComponentWithClass(input, 'component-input');
+    assert.isDefined(renderedInput);
+
+  });
+
+  it('should render tel input', function() {
+    var input = TestUtils.renderIntoDocument(
+      <InputView type="tel" />
+    );
+
+    var renderedInput = TestUtils.findRenderedDOMComponentWithClass(input, 'component-input');
+    assert.isDefined(renderedInput);
+
+  });
+
+  it('should render email input', function() {
+    var input = TestUtils.renderIntoDocument(
+      <InputView type="email" />
+    );
+
+    var renderedInput = TestUtils.findRenderedDOMComponentWithClass(input, 'component-input');
+    assert.isDefined(renderedInput);
+
+  });
+
+  it('should render number input', function() {
```

Comment:
as well as asserting that these actually render the component could we assert that they render with the correct type attribute?